community/kernel/src/test/java/org/neo4j/kernel/impl/util/dbstructure/DbStructureArgumentFormatterTest.java

```diff
@@ -72,8 +72,16 @@ public void shouldFormatIndexDescriptors()
     @Test
     public void shouldFormatUniquenessConstraints()
     {
-        assertEquals( "ConstraintDescriptorFactory.uniqueForLabel( 23, 42 )", formatArgument(
-                ConstraintDescriptorFactory.uniqueForSchema( SchemaDescriptorFactory.forLabel( 23, 42 ) ) ) );
+        assertEquals( "ConstraintDescriptorFactory.uniqueForLabel( 23, 42 )",
+                formatArgument(
+                        ConstraintDescriptorFactory.uniqueForLabel( 23, 42 ) ) );
+    }
+
+    @Test
+    public void shouldFormatCompositeUniquenessConstraints()
+    {
+        assertEquals( "ConstraintDescriptorFactory.uniqueForLabel( 23, 42, 43 )",
+                formatArgument( ConstraintDescriptorFactory.uniqueForLabel( 23, 42, 43 ) ) );
```

Comment:
No. You have to remember that `ConstraintDescriptorFactory.uniqueForLabel( 23, 42, 43 ) )` is computed to `ConstraintDescriptor` before it is sent down to the method, so whenever we create it via a schema or with the labels directly does not matter. Thus I choose to do it this way, since it matches the result better :)